src/cuga/backend/slash_commands/command_resolver.py

```
@@ -0,0 +1,131 @@
+
+    name: str
+    kind: str
+    description: str
+    score: float
+
+
+def _cosine(a: Sequence[float], b: Sequence[float]) -> float:
+    """Cosine similarity between two vectors; 0.0 for any zero-norm input."""
+    if not a or not b:
+        return 0.0
+    dot = sum(x * y for x, y in zip(a, b))
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)